### PR TITLE
Add font support for chinese, japanese, korean.

### PR DESCRIPTION
**Pull request by awaysoft**
Added some character to DejaVuSans Font, so Language List can show CJK names.
Modify code to let ddnet choose DejavuWenQuanYiMicroHei when user choose CJK language, this font is merge from Dejavu and WenQuanYiMicroHei.

### CODE REVIEW

File: src/game/client/components/menus_settings.cpp [hunk 1]
```diff
@@ -2,6 +2,7 @@
 /* If you are missing that file, acquire a complete release at teeworlds.com.                */
 
 #include <base/tl/string.h>
+#include <string.h>
```
Comment (H-M-H) [inline src/game/client/components/menus_settings.cpp]: Better use the stringfunctions provided here: https://github.com/ddnet/ddnet/blob/master/src/base/system.h#L740-L1030

Comment (awaysoft) [inline src/game/client/components/menus_settings.cpp]: Replaced it

**Comment by awaysoft** [comment]
@heinrich5991 fix compile warning.

**Comment by heinrich5991** [comment]
Thanks, looks fine to me now.

**Pull request merged**